src/app/pages/account/users/user-list-details/user-list-details.component.html

```diff
@@ -0,0 +1,37 @@
+<ng-container matColumnDef="expandedDetail">
+  <td mat-cell *matCellDef="let user; dataSource: dataSource" [attr.colspan]="colspan">
+    <ix-table-expandable-row [data]="getDetails(user)" [isExpanded]="expandedRow === user">
```

Comment:
Not sure about this, but initially I thought we'd only move what's inside of cell definition (rows 3 - 28) into `user-list-details` and keep rest (table related stuff) in parent component. This seems to fit single responsibility principle better, where parent component deals with tables and details component deals with a specific row.
But I am not super sure on this one, so we can keep current approach as well.